elevation_mapping/src/ElevationMap.cpp

```diff
@@ -267,6 +267,13 @@ bool ElevationMap::fuse(const grid_map::Index& topLeftIndex, const grid_map::Ind
     // http://www.reid.ai/2012/09/chi-squared-distribution-table-with.html
     const double uncertaintyFactor = 2.486; // sqrt(6.18)
     Eigen::EigenSolver<Eigen::Matrix2d> solver(covarianceMatrix);
+
+    // Validate whether the eigensolver did succeed
+    if (solver.info() == Eigen::NumericalIssue) {
+      ROS_ERROR_STREAM("EigenSolver NumericalIssue when fusing the map, covarianceMatrix: " << covarianceMatrix);
```

Comment:
The error message tells you that there was an error but not how the program deals with it. Maybe you could add some information that it is skipping the current area. Also, I would add a newline before printing the covariance matrix, for readability.